package.json

```diff
@@ -9,7 +9,7 @@
     "eslint-plugin-react-compiler": "19.1.0-rc.2",
     "eslint-plugin-react-hooks": "5.2.0",
     "globals": "16.3.0",
-    "typescript-eslint": "8.39.1"
+    "typescript-eslint": "8.40.0"
```

Comment:
Wait for https://github.com/open-turo/eslint-config-typescript/pull/424 to build tomorrow AM so that all versions are in sync